models/question.ts

```diff
@@ -1,10 +1,17 @@
 import { Answer } from './answer'
 
+export enum QuestionType {
+  'RESULT' = 'RESULT',
+  'CONTINUE' = 'CONTINUE',
+  'RADIO' = 'RADIO',
+  'TEXT' = 'TEXT',
```

Comment:
I am not frontend but is the enum supposed to be smth like
```suggestion
export enum QuestionType {
   RESULT  = 'RESULT',
   CONTINUE = 'CONTINUE',
   RADIO  = 'RADIO',
   TEXT  = 'TEXT',
```